app/controllers/admin/stats/top_balances_controller.rb

```diff
@@ -3,11 +3,11 @@ def index
     @limit = params[:limit].present? ? params[:limit].to_i : 25
     @result = JSON.parse(case params[:type]
     when Eyeshade::TopBalances::CHANNEL
-      Rails.cache.fetch(Cache::EyeshadeStatsJob::EYESHADE_TOP_CHANNEL_BALANCES) || []
+      Rails.cache.fetch(Cache::EyeshadeStatsJob::EYESHADE_TOP_CHANNEL_BALANCES) || "{}"
```

Comment:
There should be `"[]"`

Reply:
https://github.com/brave-intl/publishers/pull/1663/commits/5cfd73e7dc3e851dc331bb9194ecb0abd505ae08